common/static/common/js/components/views/paging_header.js

```diff
@@ -1,113 +1,64 @@
-                    messageHtml = this.messageHtml();
-                this.$el.html(_.template(paging_header_template, {
-                    messageHtml: messageHtml
-                }));
-                this.$(".previous-page-link").toggleClass("is-disabled", currentPage === 0).attr('aria-disabled', currentPage === 0);
-                this.$(".next-page-link").toggleClass("is-disabled", currentPage === lastPage).attr('aria-disabled', currentPage === lastPage);
-                return this;
+            initialize: function (options) {
+                this.collections = options.collection;
+                this.itemDisplayNameSingular = options.itemDisplayNameSingular || gettext('item');
+                this.itemDisplayNamePlural = options.itemDisplayNamePlural || gettext('items');
+                this.collection.bind('add', _.bind(this.render, this));
+                this.collection.bind('remove', _.bind(this.render, this));
+                this.collection.bind('reset', _.bind(this.render, this));
             },
 
-            messageHtml: function() {
-                var message = '';
-                var asset_type = false;
-                if (this.view.collection.assetType) {
-                    if (this.view.collection.sortDirection === 'asc') {
-                        // Translators: sample result:
-                        // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added ascending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s ascending');
-                    } else {
-                        // Translators: sample result:
-                        // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added descending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s descending');
-                    }
-                    asset_type = this.filterNameLabel();
+            render: function () {
+                var message,
+                    start = this.collection.start,
+                    end = start + this.collection.length,
+                    num_items = this.collection.totalCount;
+                if (!this.collection.hasPreviousPage() && !this.collection.hasNextPage()) {
+                    // One page of results
+                    message = interpolate(
+                        ngettext(
+                            /*
+                             * Translators: 'num_items' is the number of items that the student sees
+                             * 'item_display_name_singular' is the translated singular form of the item name
+                             * 'item_display_name_plural' is the translated plural form of the item name
+                             */
+                            'Currently viewing %(num_items)s %(item_display_name_singular)s',
```

Comment:
This will never display "Currently viewing all 1 item", since we split based on `num_items`.

I'll defer to @catong's judgement on whether we want to ever display this message in the case where there's one item to display. I think this is a rare case for a paginated collection.